### PR TITLE
Edit Note screen check mark text changed to "Save"

### PR DESCRIPTION
**Pull request by johndem**
Needs to be translated to all languages(currently valid only on english
and greek) - Issue #3894

### CODE REVIEW

File: AnkiDroid/src/main/res/values-el/02-strings.xml [hunk 1]
```diff
@@ -93,6 +93,7 @@
   <string name="saving_facts">Αποθήκευση σημείωσης</string>
   <string name="saving_model">Saving model</string>
   <string name="add">Προσθήκη</string>
+  <string name="save">Αποθήκευση</string>
```
Comment (eginhard) [inline AnkiDroid/src/main/res/values-el/02-strings.xml]: Thanks! No need to add the translation here, they'll be pulled from the [CrowdIn](https://crowdin.com/project/ankidroid) platform.

**Comment by timrae** [comment]
https://github.com/ankidroid/Anki-Android/wiki/Contributing#translate-ankidroid

**Comment by hssm** [comment]
I think there was a bit of a misunderstanding with this PR. It *does* fix #3894. The translated string was just an extra. The rest of it was okay to merge. I can't re-open this PR now because the same branch is being used in https://github.com/ankidroid/Anki-Android/pull/4050.

Sorry for the trouble. Would you be able to create a new branch with this change again (minus the translation)?

**Comment by johndem** [comment]
Yes I will.

**Comment by timrae** [comment]
oops my bad, thanks @hssm

**Pull request closed without merging**